namespaces/nsenter/nsenter.c

```diff
@@ -151,6 +151,17 @@ void nsenter()
 			exit(1);
 		}
 	}
+	// blocking until the parent placed the process inside correct cgroups.
+	unsigned char s;
+	ssize_t ret;
+	do {
+		ret = read(3, &s, 1);
+	} while (ret == -1 && errno == EINTR);
+	if (ret != 1 || s != '1') {
+		fprintf(stderr,
```

Comment:
You should probably rebase and use the pr_perror macro from this commit https://github.com/docker/libcontainer/commit/35ae1c48710ff5a4db20645bc98c719cfb695b9a